plugins/lang/statscfg.txt

```diff
@@ -320,15 +320,7 @@ ST_SAY_REPORT = Say /report
 ST_SAY_SCORE = Say /score
 ST_SAY_TOP15 = Say /top15
 ST_SAY_STATS = Say /stats
-ST_SPEC_RANK = Spec. Rank Info
-ST_BOMB_PLANTED_SOUND = Bomb Planted Sound
-ST_BOMB_DEF_SUCC_SOUND = Bomb Defuse Succ. Sound
-ST_BOMB_DEF_FAIL_SOUND = Bomb Def. Failure Sound
-ST_BOMB_CD_HUD = Bomb Countdown Hud
-ST_LAST_MAN_SOUND = Last Man Sound
-ST_LAST_MAN_HEALTH = Last Man Health
-ST_HE_KILL_SOUND = Grenade Kill Sound
-ST_HE_SUICIDE_SOUND = Grenade Suicide Sound
```

Comment:
Why did you remove these?

Reply:
https://github.com/alliedmodders/amxmodx/commit/01d20ee7f54d4dd86f53f192402c0ab8090ea04d

Reply:
> Why did you remove these?

https://github.com/alliedmodders/amxmodx/commit/01d20ee7f54d4dd86f53f192402c0ab8090ea04d